src/nunit.analyzers/Constants/ConstActualValueUsageAnalyzerConstants.cs

```diff
@@ -1,10 +1,12 @@
+using System;
+
 namespace NUnit.Analyzers.Constants
 {
     internal static class ConstActualValueUsageAnalyzerConstants
     {
-        public const string Title = "The actual value should not be a constant.";
-        public const string Message = "The actual value should not be a constant - " +
-            "perhaps the actual value and the expected value have switched places.";
+        public const string Title = "The actual value should not be a constant";
+        public const string Message = "The actual value should not be a constant" +
+            "perhaps the actual value and the expected value have switched places";
```

Comment:
This will result in the word `constantperhaps`. I think we should either reintroduce the `-` or rephrase the message into two sentences, separated with e `.`.

Reply:
Reintroduced the ` - `.